crates/spl_network/src/message_filter.rs

```diff
@@ -32,9 +32,10 @@ impl MessageFilter {
             IncomingMessage::GameController(source_address, message) => Some(
                 IncomingMessage::GameController(*source_address, message.clone()),
             ),
-            IncomingMessage::Spl(message) if message.player_number != *context.player_number => {
-                Some(IncomingMessage::Spl(*message))
-            }
+            IncomingMessage::Spl(
+                message @ HulkMessage::Striker(StrikerMessage { player_number, .. })
+                | message @ HulkMessage::VisualReferee(VisualRefereeMessage { player_number, .. }),
```

Comment:
Is there a syntactic way to deduplicate the `message @` tag?

Reply:
`message @ (...)` works to, done